tests/providers/google/marketing_platform/operators/test_analytics_system.py

```diff
@@ -17,18 +17,38 @@
 
 import pytest
 
+from airflow.providers.google.marketing_platform.example_dags.example_analytics import BUCKET, BUCKET_FILENAME
 from tests.providers.google.cloud.utils.gcp_authenticator import GMP_KEY
 from tests.test_utils.gcp_system_helpers import MARKETING_DAG_FOLDER, GoogleSystemTest, provide_gcp_context
 
+DATA = """pagePath,dimension1
+how_to_make_pizza,1
+how_to_make_doughnuts,2
+how_to_build_a_treehouse,3
+how_to_make_paper_airplanes,4
+"""
+
+
 # Required scopes
 SCOPES = [
```

Comment:
Is it working without this scope `https://www.googleapis.com/auth/analytics.readonly`?

Reply:
Yes, the documentation mentions using "at least one"

Reply:
Ok :)